extensionsGallery.json

```diff
@@ -4610,7 +4610,7 @@
 							"properties": [
 								{
 									"key": "Microsoft.VisualStudio.Code.Engine",
-									"value": ">=1.58.0"
+									"value": ">=1.67.0"
```

Comment:
This should go in the RC branch, right? Stable can't support it if this is accurate.

Reply:
Updated target branch to rc1

Reply:
Let me update the icon first

Reply:
Never mind, didn't need to update it